functions/brinks/src/middleware/stripeEventEnd.js

```diff
@@ -0,0 +1,31 @@
+import debug from 'debug';
+import * as Sentry from '@sentry/node';
+
+const dlog = debug('that:api:functions:brinks:stripeEventEndMw');
+
+export default function stripeEventEnd(req, res) {
+  dlog('stripeEventEnd called');
+  const { thatBrinks } = req;
+  thatBrinks.stages.push('stripeEventEnd');
+
+  // Check for a status being set.
```

Comment:
Might be worth cleaning these up a bit as it current reads like your notes.

Reply:
cleaned up in brinks: ac099c2